Frontend/src/database/Database.js

```diff
@@ -116,6 +116,17 @@ class Database {
     );
   }
 
+ countDocs(selectors) {
```

Comment:
This new function looks great, could you also call it in line 173 to prevent duplicate code?

Reply:
but it's not async at the moment, and on line 173 it's loaded async, or am I misunderstanding the code?

Reply:
included it now